src/static/css/index.css

```diff
@@ -639,3 +641,9 @@ p {
     margin: 0;
   }
 }
+
+@media (max-width: 320px) {
```

Comment:
Below 315px this starts wrapping anyway:

![image](https://user-images.githubusercontent.com/1120896/68541150-d68c3000-0350-11ea-9c27-6622e8b77ca2.png)